projects/demo/src/modules/components/input-time/examples/5/index.ts

```diff
@@ -0,0 +1,32 @@
+import {Component} from '@angular/core';
+import {FormControl, FormGroup} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TUI_INPUT_TIME_DEFAULT_OPTIONS, TUI_INPUT_TIME_OPTIONS} from '@taiga-ui/kit';
+
+@Component({
+    selector: 'tui-input-time-example-5',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+    providers: [
+        {
+            provide: TUI_INPUT_TIME_OPTIONS,
+            useValue: {
+                ...TUI_INPUT_TIME_DEFAULT_OPTIONS,
+                mode: 'HH:MM',
+                maxValues: {HH: 11, MM: 59},
```

Comment:
The solution could've been better if I spent more time playing with the mask but since this field is time-only, I suppose the current implementation is somewhat acceptable.

Reply:
Ok, let's keep the am/pm issue open then to improve this later.